pkg/virt-controller/watch/vmi.go

```diff
@@ -2353,12 +2363,17 @@ func (c *VMIController) getVolumePhaseMessageReason(volume *virtv1.Volume, names
 	return virtv1.VolumePending, PVCNotReadyReason, "PVC is in phase Lost"
 }
 
-func (c *VMIController) updateMultusAnnotation(namespace string, interfaces []virtv1.Interface, networks []virtv1.Network, pod *k8sv1.Pod) error {
+func (c *VMIController) updateMultusAnnotation(vmi *virtv1.VirtualMachineInstance, interfaces []virtv1.Interface, networks []virtv1.Network, pod *k8sv1.Pod) error {
```

Comment:
If you need also de vmi name you can pass `NamespacedName` https://pkg.go.dev/k8s.io/apimachinery/pkg/types#NamespacedName to pass the vmi key, or go for the gold and remove `interfaces` and `networks` and read it from the vmi too.

Reply:
We can't remove the networks, because they must be calculated outside (was before this PR)
(for example calculated by `CalculateInterfacesAndNetworksForMultusAnnotationUpdate`)
i will just pass name and namespace, no need to compose and then break to two parts